ppr-ui/src/components/mhrRegistration/HomeOwners/HomeOwnersTable.vue

```diff
@@ -628,11 +634,17 @@ export default defineComponent({
 @import '@/assets/styles/theme.scss';
 
 .home-owners-table ::v-deep {
-  .executor-icon {
+  .person-executor-icon {
     margin-top: -3px !important;
     height: 22px !important;
     width: 22px !important;
   }
+  .business-executor-icon {
+    margin-top: -8px !important;
+    margin-left: -4px !important;
+    height: 29px !important;
+    width: 28px !important;
+  }
```

Comment:
the svgs a little different in size? 
Kind of annoying i imagine 🙄

Reply:
Very annoying

Reply:
ones square and the other isn't, thats why all the gross css is needed

Reply:
Oof